schemas/config-schema.json

```diff
@@ -205,11 +204,11 @@
                             },
                             "metadata": {
```

Comment:
Does this need to change with `meta_data`?

Reply:
think we can still call this metadata. just if people don't define it, we need to look at a `meta_data` folder.